crates/egui/src/context.rs

```diff
@@ -1905,6 +1905,11 @@ impl Context {
         self.memory_mut(|mem| mem.areas_mut().move_to_top(layer_id));
     }
 
+    /// Retrieve the `LayerId` of the top level windows.
+    pub fn get_top_layer_id(&self) -> Option<LayerId> {
```

Comment:
```suggestion
    /// Retrieve the [`LayerId`] of the top level windows.
    pub fn top_layer_id(&self) -> Option<LayerId> {
```

https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter